### ISSUE TITLE
IOS VoiceOver. Impossible to get an access to tab content

### ISSUE THREAD

**Issue by bogomya** [davidtheclark/react-aria-tabpanel#17]
There is no ways to get an access to tab content from any IOS device.

Action Performed:
1. Using iOS VO+Safari open http://davidtheclark.github.io/react-aria-tabpanel/demo/
2. Go to Stateful demo
3. Swipe left or right to change focus between tabs
4. Tab content is unreachable from any tab, except the last one.

How it can be resolved:
https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html

**Comment by davidtheclark** [comment]
@bogomya: What does that link suggest that is missing from the current implementation? Open to any PR fixing the accessibility!

**Comment by sloyless** [comment]
Also encountering this issue with Voiceover on iOS. JAWS and NVDA work fine, as well as VO on desktop. Content is completely unfocusable.
